From 3db8d5e750dab5f565433d695121b9782a6d4c04 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 7 Dec 2021 01:20:56 +0800
Subject: [PATCH 05/33] Revert "drm/amdgpu: fix set scaling mode Full/Full
 aspect/Center not works on vga and dvi connectors"

This reverts commit 04c586a601dc0acb8e739205db350ab660dffc4f.
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_connectors.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_connectors.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_connectors.c
index 0de66f59adb8..b9c11c2b2885 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_connectors.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_connectors.c
@@ -827,7 +827,6 @@ static int amdgpu_connector_vga_get_modes(struct drm_connector *connector)
 
 	amdgpu_connector_get_edid(connector);
 	ret = amdgpu_connector_ddc_get_modes(connector);
-	amdgpu_get_native_mode(connector);
 
 	return ret;
 }
-- 
2.27.0

